data/Makefile.am

```diff
@@ -61,6 +61,7 @@ filetypes = \
 	filedefs/filetypes.ruby \
 	filedefs/filetypes.rust \
 	filedefs/filetypes.Scala.conf \
+	filedefs/filetypes.Swift.conf \
```

Comment:
not very important, but we sort them alphabetically and case insensitively, so it should go after `filetypes.sql`